packages/web/src/common/store/upload/sagas.js

```diff
@@ -1015,18 +1012,25 @@ function* uploadSingleTrack(track) {
           throw new Error(`Could not confirm upload single track ${trackId}`)
         }
 
-        // getting track immediately after acdc confirms
-        // can 404 because it's not indexed yet
-        yield delay(1500)
-
-        return yield apiClient.getTrack({
-          id: trackId,
-          currentUserId: userId,
-          unlistedArgs: {
-            urlTitle: formatUrlName(track.metadata.title),
-            handle
+        let retries = 30
```

Comment:
this is all post confirmation, right? so we're not depending on upload or confirmation time to fit within the retry window.

Reply:
no, the upload has already succeeded, this is purely for indexing

Reply:
brilliant. Sounds good to me!